src/components/Charts/RowChart.jsx

```diff
@@ -151,6 +151,7 @@ export class RowChart extends React.Component {
       this.props.total > 0 &&
       <div className="row-chart-section">
         <h3>{ this.props.title }</h3>
+        <p>{ this.props.helperText }</p>
```

Comment:
you can add helperText to the RowChart.propTypes and make it optional if you want.  Not really necessary, but we have that in this component.